src/Command/ModelCommand.php

```diff
@@ -1382,4 +1383,28 @@ public function bakeTest(string $className, Arguments $args, ConsoleIo $io): voi
         );
         $test->execute($testArgs, $io);
     }
+
+    /**
+     * @param array<string, mixed> $data
+     * @param string $entity
+     * @param string $namespace
+     * @return array<string, class-string>
+     */
+    protected function enums(array $data, string $entity, string $namespace): array
+    {
+        $fields = $data['fields'];
+        $enumClassNamespace = $namespace . '\Model\Enum\\';
+
+        $enums = [];
+        foreach ($fields as $field) {
```

Comment:
Don't we have a way to iterate through only sting and integer fields?

Reply:
I found a way

Reply:
Being pedantic about the db column type will just increase support questions :) I am sure lot of people don't bother and just use integer field.

Reply:
Fair enough